scripts/updatebreaches.js

```diff
@@ -0,0 +1,97 @@
+"use strict";
+
+const got = require("got");
+
+const AppConstants = require("../app-constants");
+const HIBP = require("../hibp");
+
+
+const FX_RS_RECORDS = `${AppConstants.FX_REMOTE_SETTINGS_WRITER_SERVER}/buckets/main-workspace/collections/fxmonitor-breaches/records`;
+const FX_RS_COLLECTION = `${AppConstants.FX_REMOTE_SETTINGS_WRITER_SERVER}/buckets/main-workspace/collections/fxmonitor-breaches`;
+const FX_RS_PREVIEW_COLLECTION = `${AppConstants.FX_REMOTE_SETTINGS_WRITER_SERVER}/buckets/main-preview/collections/fxmonitor-breaches`;
+const FX_RS_USERNAME = AppConstants.FX_REMOTE_SETTINGS_USERNAME;
+const FX_RS_PASSWORD = AppConstants.FX_REMOTE_SETTINGS_PASSWORD;
+
+
+if (!FX_RS_USERNAME || !FX_RS_PASSWORD) {
+  console.error("updatebreaches requires FX_REMOTE_SETTINGS_USERNAME and FX_REMOTE_SETTINGS_PASSWORD.");
+  process.exitCode = 1;
+  return;
+}
+
+
+function whichBreachesAreNotInRemoteSettingsYet(remoteSettingsBreachesSet, hibpBreaches) {
+  const newBreaches = [];
+  for (const breach of hibpBreaches) {
+    if (breach.IsSpamList || breach.IsRetired || !breach.IsVerified || !breach.Domain) {
+      continue;
+    }
+    if (remoteSettingsBreachesSet.has(breach.Name)) {
+      continue;
+    }
+    newBreaches.push(breach);
+  }
```

Comment:
Do we need to `return newBreaches;` somewhere?
I see we have a `const newBreaches = whichBreachesAreNotInRemoteSettingsYet(remoteSettingsBreachesSet, hibpBreaches);` on L45 below, but not sure how that wouldn't always be `undefined`, unless I'm missing a trick somewhere.

Reply:
Good catch. I caught that while addressing @nhnt11's comments too.

Reply:
Couldn't all this written like:
```js
const newBreaches = hibpBreaches.filter(({ Name }) => !remoteSettingsBreachesSet.has(Name))
```
?